config/manifest-helpers.js

```diff
@@ -9,7 +9,7 @@ function getAppManifests(root) {
       const manifest = require(file);
 
       manifest.filePath = file;
-      manifest.entryFile = path.relative(root, path.join(path.dirname(file), manifest.entryFile));
+      manifest.entryFile = path.resolve(root, path.join(path.dirname(file), manifest.entryFile));
```

Comment:
I found that we need an absolute path instead of relative if we're executing from outside of the project root.